.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ commands:
       - run:
           name: 'Run fast tests'
           command: |
-            yarn test:fast --reporter=junit --outputFile="./reports/out/test_output.xml"
+            yarn test:fast --reporter=default --reporter=junit --outputFile="./reports/out/test_output.xml"
```

Comment:
Using the `default` reporter both gives us a clearer idea of the failures in CI and prevents us from hitting the 10-minute CircleCI `no output` timeout.